articles/Silo_Finance.md

```diff
@@ -0,0 +1,388 @@
+
+For bridge assets (ETH or XAI), the naming convention is slightly different, i.e._ sBridge-ABC_, whereby ABC represents the name of the silo that ETH or XAI is deposited into. For instance, when depositing the bridge asset ETH into the LINK silo, the lender in return receives _sETH-LINK_ or _spETH-LINK_.
+
+
+## Debt Tokens (dTokens)
+
+Conversely, [dTokens](https://silopedia.silo.finance/borrow/debt-tokens-dtokens) are ERC-20 tokens representing a borrowed asset. They are minted when an asset is borrowed and burned when the loan is paid back. Unlike _sTokens_, _dTokens_ cannot be transferred out of a wallet. For bridge assets, the borrowed token is represented again as _dBridge-ABC_ (e.g. dETH-UNI).
+
+
+## Conclusion
+
+In summary, Silo introduces a few innovations and a new money market primitive aiming to make lending markets less risky, while simultaneously improving market access for long-tail assets. The ability to protect tokens from borrowers can be a very interesting feature for other DAOs to deploy their tokens and borrow tokens against them.
+
+The risk reduction, however, has one trade-off with the current design: Borrowing at Silo can initially result in up to six transactions. First, <allow deposit> and <deposit> of the collateral, then <borrow> the bridge asset, then again <allow deposit> and <deposit> the bridge asset, before finally the targeted asset can be <borrowed>. Reversing it will incur another four transactions.
+The image below displays an example of the Silo dashboard for depositing APE, borrowing ETH, then depositing ETH to borrow CVX.
+ 
+![Silo_Dashboard](https://user-images.githubusercontent.com/89845409/200535504-2d97c75a-aeb7-4025-8c3c-3c31626f8288.png)
+
+(source: [Blog](https://medium.com/silo-protocol/silo-is-live-in-beta-mainnet-94b6b0164258))
+
+This splits the risk for the user into two. For instance, if CVX moons to the point that the user might get liquidated, the originally deposited APE is still safe. On the other hand, this can be rather costly, especially on the Ethereum mainnet, and might be a blocker for smaller accounts to use Silo as intended.
+The team is working on new features to improve this, but it will take some time. The introduction of XAI as a bridge asset might improve this too, as most users are typically looking to borrow stable assets. They can now do that where XAI is enabled as a bridge asset, but more on that in the next section.
```

Comment:
any link to blog post about features to improve this? Or any info we can give about how they intend to improve?

Reply:
Yep Aiham mentioned something. I added this -> "When talking to the team they revealed that they are working on a swap-like feature to improve this. And the long-term vision is to enable a unified health-factor, so all isolated positions are unified and managed as one, but it will take some time."